tests/Util/fixtures/add_many_to_many_relation/User_simple_inverse.php

```diff
@@ -53,10 +53,8 @@ public function addRecipe(Recipe $recipe): self
 
     public function removeRecipe(Recipe $recipe): self
     {
-        if ($this->recipes->contains($recipe)) {
-            $this->recipes->removeElement($recipe);
-            $recipe->removeFood($this);
-        }
+        $this->recipes->removeElement($recipe);
+        $recipe->removeFood($this);
```

Comment:
I think to mirror the behavior from before this should be

```php
if ($this->recipes->removeElement($recipe)) {
    $recipe->removeFood($this);
}
```

instead?

Reply:
Fixed, thanks